### PR TITLE
Update Order Page View.md

### PR DESCRIPTION
**Pull request by MatShir**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Multistore in the order page
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/21706

### CODE REVIEW

File: back-office/orders/orders/Order Page View.md [hunk 1]
```diff
@@ -544,5 +546,14 @@ Steps to display:
 It displays the date and from where the user came from.
 
 
-## VI. Links
+## VII. Links
+
+## VIII. Multistore
+
+An order from a specific shop can be edited from any shops. Even if they are separeted from differents groups.
```
Comment (marionf) [inline back-office/orders/orders/Order Page View.md:553]:
```suggestion
An order made from a specific shop can be edited from any shops. Even if the shop where the order is edited isn't in the same group as the shop where the order was made.
```

File: back-office/orders/orders/Order Page View.md [hunk 2]
```diff
@@ -544,5 +546,14 @@ Steps to display:
...
+
+In a multistore context, the next order and previous buttons work depending on the view of the shop or the group of shops the Back-Office is set.
```
Comment (marionf) [inline back-office/orders/orders/Order Page View.md:555]:
```suggestion
In a multistore context, the next and previous buttons for navigating between orders work according to the selected context in the back-office (specific shop, group of shops or all shops)
```

File: back-office/orders/orders/Order Page View.md [hunk 3]
```diff
@@ -544,5 +546,14 @@ Steps to display:
...
+If the order listing is filtered by a multistore group view then the user can navigate between orders in the order view page only from this shop group.
```
Comment (marionf) [inline back-office/orders/orders/Order Page View.md:556]:
```suggestion
If a specific shop group is selected as context in the order list, then the user can navigate between orders belonging only to this shop group in the order view page.
```

File: back-office/orders/orders/Order Page View.md [hunk 4]
```diff
@@ -544,5 +546,14 @@ Steps to display:
...
+If the order listing is filtered by a specific shop from the multistore setting then the user can navigate between orders in the order view page only from this shop.
```
Comment (marionf) [inline back-office/orders/orders/Order Page View.md:557]:
```suggestion
If a specific shop is selected as context in the order list, then the user can navigate between orders belonging only to this shop in the order view page.
```

File: back-office/orders/orders/Order Page View.md [hunk 5]
```diff
@@ -544,5 +546,14 @@ Steps to display:
...
+And if the order listing is set in "All shop" in the multistore setting then the user can navigate between orders in the order view page from all the shops.
```
Comment (marionf) [inline back-office/orders/orders/Order Page View.md:558]:
```suggestion
If "All shops" is selected as context in the order list, then the user can navigate between orders from all the shops in the order view page.
```

**Review by marionf — APPROVED** [review]

**Comment by MatShir** [comment]
Need merge for other PR

**Pull request merged**